### PR TITLE
Task 3. ShoppingList app with MVC (add with limit 10 items, remove, show all).

### PR DESCRIPTION
**Pull request by badpanda13**
Hello. Could you review my shopping list app, please. Thank you a lot.

### CODE REVIEW

File: task 3/Task 3/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//
+//  Created by Belka on 3/4/20.
+//  Copyright © 2020 Tanya. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
```
Comment (dersim-davaod) [inline task 3/Task 3/AppDelegate.swift]: Remove empty lines?

File: task 3/Task 3/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+    }
+
+    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+        // Called when the user discards a scene session.
+        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+    }
+
```
Comment (dersim-davaod) [inline task 3/Task 3/AppDelegate.swift]: Remove blank lines.
Please double-check the rest of code.

File: task 3/Task 3/ShoppingListController.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    func remove(at index: Int) -> Bool {
+        if index < 0 || index >= shoppingList.count  {
+            return false
+        }
+        shoppingList.remove(at: index)
+        
+        return true
+    }
+    
+    func getShoppingList() -> [String]{
```
Comment (dersim-davaod) [inline task 3/Task 3/ShoppingListController.swift]:
```suggestion
    func getShoppingList() -> [String] {
```

I saw commit where you have added SwiftLint. Probably you should improve its configuration.

File: task 3/Task 3/AppDelegate.swift (hunk 1 above)
Comment (badpanda13) [reply task 3/Task 3/AppDelegate.swift]: I`ll remove empty lines after it, thank you.

File: task 3/Task 3/AppDelegate.swift (hunk 2 above)
Comment (badpanda13) [reply task 3/Task 3/AppDelegate.swift]: If you about empty lines - OK, I`ll remove it and I`ll check all code for this.

File: task 3/Task 3/ShoppingListController.swift (hunk 3 above)
Comment (badpanda13) [reply task 3/Task 3/ShoppingListController.swift]: OK, thank you. I`ll configure swiftLint for this and fix it.

**Comment by badpanda13** [comment]
My model layer contains ShoppingListService and ShoppingListProtocol, because they work with data  in its own domain and return results for controller layer. As I understand, there aren`t structures specifically for data (persistence) in this app, only array of Strings.

**Pull request merged**